src/components/TabletApp/LeftSide/LeftSide.tsx

```diff
@@ -1,13 +1,73 @@
-import React, { useState } from "react";
+import React, { useRef, useState } from "react";
 import { Box } from "@mui/system";
-import { Typography, Button } from "@mui/material";
+import { Button, Paper, ThemeProvider, Typography } from "@mui/material";
 import UpcomingCards from "./UpcomingCards/UpcomingCards";
-import CalendarMonthIcon from "@mui/icons-material/CalendarMonth";
+import "./UpcomingCardsScrollCSS.css";
+import Dayjs from "dayjs";
+import GlobalTheme from "../../../context/GlobalThemes";
+
 interface iLeftSide {
-    roomName: string;
+    name: string | undefined;
+    meetings:
+        | {
```

Comment:
Create a new type for this meeting like Meeting where you specify the fields, don't inline them here